tools/ci_build/github/linux/docker/inference/aarch64/default/cpu/scripts/install_python.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -e -x
+os_major_version=$(tr -dc '0-9.' < /etc/redhat-release |cut -d \. -f1)
+echo "installing Python for CentOS version : $os_major_version"
```

Comment:
This docker file is meant to be used for build Nuget packages and anything that is not python. 
Use a separated docker file for python. 
The python one should be based on manylinux.